spring-aop/README.md

```diff
@@ -6,11 +6,9 @@ This module contains articles about Spring aspect oriented programming (AOP)
 
 - [Implementing a Custom Spring AOP Annotation](https://www.baeldung.com/spring-aop-annotation)
 - [Intro to AspectJ](https://www.baeldung.com/aspectj)
-- [Spring Performance Logging](https://www.baeldung.com/spring-performance-logging)
 - [Introduction to Spring AOP](https://www.baeldung.com/spring-aop)
 - [Introduction to Pointcut Expressions in Spring](https://www.baeldung.com/spring-aop-pointcut-tutorial)
 - [Introduction to Advice Types in Spring](https://www.baeldung.com/spring-aop-advice-tutorial)
-- [When Does Java Throw UndeclaredThrowableException?](https://www.baeldung.com/java-undeclaredthrowableexception)
-- [Get Advised Method Info in Spring AOP](https://www.baeldung.com/spring-aop-get-advised-method-info)
 - [Advise Methods on Annotated Classes With AspectJ](https://www.baeldung.com/aspectj-advise-methods)
 - [Joinpoint vs. ProceedingJoinPoint in AspectJ](https://www.baeldung.com/aspectj-joinpoint-proceedingjoinpoint)
+- More articles: [[next -->]](/spring-aop-2)
```

Comment:
I think the link needs to be relative, like: ```../spring-aop-2```

Reply:
@hkhan , since its a top level module, its fine to be absolute. Takes you to the right place :)